src/subcommandlineparser.cpp

```diff
@@ -45,7 +45,7 @@ static const std::map<QString, QList<QCommandLineOption>> options{
                       "The name does not need to be unique. "
                       "In this case, all matching objects are rendered. "
                       "Perl-like regular expressions are supported."),
-          QObject::tr("NAME"),
+          QObject::tr("NAME_REGEX"),
```

Comment:
better `NAME_PATTERN` ?

Reply:
I used regex over pattern because it's more precise in meaning. (Rationale: If someone does not know what REGEX is, they will likely not know what to do with PATTERN either, but someone who would understand REGEX, might be in doubt how to understand PATTERN - glob wildcard? regex? ...) But I'll happily let myself be convinced over PATTERN if I missed something in my rationale. :)

Reply:
Looking at the [man page of `locate`](https://linux.die.net/man/1/locate), you see both `REGEXP` and `PATTERN` being used as the variable name.
`PATTERN` only if both globbing and regex are possible and `REGEXP` if there's no option for globbing.
That's seconding your rationale, so I'm fine with that.

Reply:
Cool appreciate the research :+1: